PingPayments.PaymentsApi/Payments/Shared/V1/OrderItem.cs

```diff
@@ -7,7 +8,7 @@ public record OrderItem
     {
         public OrderItem(int amount, string name, decimal vat, Guid merchantId)
         {
-            Amount = amount;
+            Amount = amount.ToMinorCurrencyUnit();
```

Comment:
Denna luras nog, vi kanske ska ta bort denna med? Så man inte råkar skicka in ören och får ören ören

Reply:
Vi UX testar

Reply:
Yes, vi tar bort detta. Har kikat på Stripe och Square nu och man skickar in amount i minor currency unit i deras SDKs. Så det kanske är bra att följa den standarden.